src/test/java/org/vanilladb/core/server/ServerInit.java

```diff
@@ -50,6 +50,21 @@
 	private static final int LOADED_FLAG_POS = 0;
 	private static final Type LOADED_FLAG_TYPE = Type.INTEGER;
 	private static final Constant DATA_LOADED_VALUE = new IntegerConstant(1);
+	
+	public static String resetDb(Class<?> testClass) {
+		String testClassName = testClass.getName();
+		String dbName = DB_MAIN_DIR + "/" + testClassName;
+		
+		// Creates the main directory if it was not created before
+		File dbPath = new File(FileMgr.DB_FILES_DIR, DB_MAIN_DIR);
+		if (!dbPath.exists())
+			dbPath.mkdir();

```

Comment:
Would I be right in saying that "deleting DB" would fail if the base directory doesn't exist.

Reply:
Yes!

Reply:
Yes!